src/components/views/analytics/AnalyticsView.tsx

```diff
@@ -5,24 +5,51 @@ import { Alert, Box, Grid, Skeleton } from '@mui/material';
 
 import { hooks } from '../../../config/queryClient';
 import { CONTAINER_HEIGHT } from '../../../constants/style';
+import { Interval } from '../../../types/chart';
 import {
   getMembersWithMostViews,
   groupActionByTimeInterval,
-  intervals,
-  topMembersRangeOptions,
 } from '../../../utils/chart';
+import Select from '../../common/Select';
 import MostFrequentUsersChart from './MostFrequentUsersChart';
-import SelectDisplayedUsersLimit from './SelectDisplayedUsersLimit';
-import SelectInterval from './SelectInterval';
 import ViewsOverTimeChart from './ViewsOverTimeChart';
 
 const AnalyticsView = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const topMembersRangeOptions = [
+    {
+      value: '5',
+      label: 5,
+    },
+    {
+      value: '10',
+      label: 10,
+    },
+    {
+      value: '25',
+      label: 25,
+    },
+    {
+      value: 'all',
+      label: t('all'),
+    },
+  ];
+  const intervals: Interval[] = [
+    { label: t('day'), value: 'day', groupBy: 'yyyy-MM-dd' },
+    {
+      label: t('week'),
+      value: 'week',
+      groupBy: 'yyyy-ww',
+    },
+    { label: t('month'), value: 'month', groupBy: 'yyyy-MM' },
+  ];
```

Comment:
It is good to have these defined in this file, but you can safely move them out of the component as they will not change. And you should translate the label when you use it.